[INFO 22:19:09] my_main *******************
[INFO 22:19:09] my_main WANDB RUN ID:
[INFO 22:19:09] my_main waaleik8
[INFO 22:19:09] my_main *******************
/home/ubuntu/miniconda3/envs/epymarl/lib/python3.10/site-packages/sacred/stdout_capturing.py:179: UserWarning: tee_stdout.wait timeout. Forcibly terminating.
  warnings.warn("tee_stdout.wait timeout. Forcibly terminating.")
/home/ubuntu/miniconda3/envs/epymarl/lib/python3.10/site-packages/sacred/stdout_capturing.py:185: UserWarning: tee_stderr.wait timeout. Forcibly terminating.
  warnings.warn("tee_stderr.wait timeout. Forcibly terminating.")
[DEBUG 22:19:11] pymarl Stopping Heartbeat
[ERROR 22:19:11] pymarl Failed after 0:00:03!
Traceback (most recent calls WITHOUT Sacred internals):
  File "/home/ubuntu/PERSONAL-DIR/UOA-NEW/human-rware/src/main.py", line 44, in my_main
    run(_run, config, _log)
  File "/home/ubuntu/PERSONAL-DIR/UOA-NEW/human-rware/src/run.py", line 67, in run
    run_sequential(args=args, logger=logger)
  File "/home/ubuntu/PERSONAL-DIR/UOA-NEW/human-rware/src/run.py", line 138, in run_sequential
    mac = mac_REGISTRY[args.mac](buffer.scheme, groups, args)
  File "/home/ubuntu/PERSONAL-DIR/UOA-NEW/human-rware/src/controllers/basic_controller.py", line 50, in __init__
    self.interactive_env = InteractiveRWAREEnv(env="rware-tiny-4ag-v2",
  File "/home/ubuntu/PERSONAL-DIR/UOA-NEW/human-rware/src/robotic_warehouse/human_play.py", line 68, in __init__
    obss, _ = self.env.reset()
  File "/home/ubuntu/miniconda3/envs/epymarl/lib/python3.10/site-packages/gymnasium/wrappers/common.py", line 400, in reset
    return super().reset(seed=seed, options=options)
  File "/home/ubuntu/miniconda3/envs/epymarl/lib/python3.10/site-packages/gymnasium/core.py", line 328, in reset
    return self.env.reset(seed=seed, options=options)
  File "/home/ubuntu/miniconda3/envs/epymarl/lib/python3.10/site-packages/gymnasium/wrappers/common.py", line 293, in reset
    return env_reset_passive_checker(self.env, seed=seed, options=options)
  File "/home/ubuntu/miniconda3/envs/epymarl/lib/python3.10/site-packages/gymnasium/utils/passive_env_checker.py", line 185, in env_reset_passive_checker
    result = env.reset(**kwargs)
  File "/home/ubuntu/miniconda3/envs/epymarl/lib/python3.10/site-packages/rware/warehouse.py", line 763, in reset
    self.render()
  File "/home/ubuntu/miniconda3/envs/epymarl/lib/python3.10/site-packages/rware/warehouse.py", line 956, in render
    return self.renderer.render(self, return_rgb_array=mode == "rgb_array")
  File "/home/ubuntu/miniconda3/envs/epymarl/lib/python3.10/site-packages/rware/rendering.py", line 129, in render
    self._draw_shelfs(env)
  File "/home/ubuntu/miniconda3/envs/epymarl/lib/python3.10/site-packages/rware/rendering.py", line 183, in _draw_shelfs
    for shelf in env.shelfs:
AttributeError: 'Warehouse' object has no attribute 'shelfs'
